lib/kubernetes-deploy/kubernetes_resource/memcached.rb

```diff
@@ -33,26 +33,22 @@ def exists?
     private
 
     def memcached_deployment_exists?
-      deployments, _err, st = kubectl.run("get", "deployments", "-l name=#{@name}", "-o=json")
+      deployment, _err, st = kubectl.run("get", "deployments", "memcached-#{@name}", "-o=json")
```

Comment:
I'm not very familiar with the memcachedbuddy code, but looking at the `ServiceName()` function it seems like there are a few ways the deployment name could be generated from the CR name. Am I wrong about that?

Reply:
You are right. This is written assuming https://github.com/Shopify/cloudbuddies/pull/446 has merged.